[12.0][MIG] shopinvader_locomotive

rousseldenis opened the pull request:
Includes :

* [x] #320 

Depends on:

* [ ] https://github.com/akretion/locomotivecms-python-client/pull/1

IMPORTANT: We removed dependency on keychain as the module seems to be deprecated in v12 : https://github.com/OCA/server-auth/pull/39

This allows to be free on the implementation on how to store identification data (and to implement a specific one on top of this).

rousseldenis committed:
[12.0][MIG] shopinvader_locomotive

sbidoul committed:
[IMP] clarify Locomotive configuration

simahawk commented on shopinvader_locomotive/data/ir_cron.xml:
```diff
@@ -9,9 +9,8 @@
+        <field name="code">model._scheduler_synchronize_currency()</field>
     </record>
 
     <record forcecreate="True" id="ir_cron_export_binding" model="ir.cron">
```
I don't get why this is needed. Even if needed, shouldn't be placed into shopinvader main module or shopinvader_search_engine?

lmignon replied:
@simahawk, you're right. This one should go into shopinvader_search_engine.

simahawk reviewed (approved):
LG. 2 things:
1. I don't get why indexes export should be triggered every day here (see comment)
2. would be nice to fast track this so that I can split the generic parts to `shopinvader_client_base` or similar ;)

lmignon commented:
>     2. would be nice to fast track this so that I can split the generic parts to `shopinvader_client_base` or similar ;)

I'll fix the build and merge once travis is :green_apple:

lmignon committed:
[FIX] requirements: Install patched version of locomotivecms-python-client
The version must be removed once https://github.com/akretion/locomotivecms-python-client/pull/1 is merged

lmignon commented:
@simahawk It's merged!
